tests/integration/companies.test.js

```diff
@@ -132,7 +155,7 @@ describe('COMPANIES ROUTES - PUT /companies/:id', () => {
       expect(response.statusCode).toBe(404);
     });
 
-    it('should return 404 if billingRepresentative is from an other company', async () => {
+    it('should return 404 if billingRepresentative is from an other company and another holding', async () => {
```

Comment:
Je trouve que cet intitulé n'est pas clair non plus. La on essaye d'editer une structure ne faisant pas partie d'une sociéte mère donc notre test est plutot "verifier que la route renvoie une 404 si le chargé de facturation ne fait pas parti de la structure" 

Je mettrais qqchose du style: "should return 404 if the company doesn't belong to any holding company and the billingRepresentative belongs to another company"

Reply:
en soit il se trouve qu'il n'est pas dans une holding mais il pourrait j'aurais tendance à mettre should return 404 if billingRepresentative is not from company nor holding